### PR TITLE
feat: improve reporter metrics

### PR DESCRIPTION
**Pull request by pasha-codefresh**

### CODE REVIEW

**Comment by pasha-codefresh** [comment]
@CodiumAI-Agent /review

**Comment by pasha-codefresh** [comment]
@CodiumAI-Agent /review

**Comment by pasha-codefresh** [comment]
@CodiumAI-Agent /review

File: event_reporter/metrics/metrics.go [hunk 1]
```diff
@@ -43,57 +45,54 @@ const (
...
 		[]string{"reporter_shard"},
 	)
-	appEventsCounter = prometheus.NewCounterVec(
+
+	enqueuedEventsCounter = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
-			Name: "cf_e_reporter_app_events_size",
-			Help: "Size of specific application events queue of taked shard.",
+			Name: "codefresh_event_reporter_enqueued_events_total",
+			Help: "Amount of specific application events queue of taken shard.",
```
Comment (zarbis) [inline event_reporter/metrics/metrics.go]: "Amount of specific application events put in the queue of a particular shard."

**Review by oleksandr-codefresh — APPROVED** [review]

File: event_reporter/metrics/metrics.go [hunk 2]
```diff
@@ -43,57 +45,54 @@ const (
...
-			Buckets: []float64{0.1, 0.25, .5, 1, 2},
-		},
-		[]string{"initiator"},
-	)
-	queueSizeCounter = prometheus.NewGaugeVec(
+	queueSizeGauge = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
-			Name: "cf_e_reporter_queue_size",
+			Name: "codefresh_event_reporter_queue_size",
 			Help: "Size of application events queue of taked shard.",
```
Comment (zarbis) [inline event_reporter/metrics/metrics.go]: "Size of application events queue of a particular shard."

File: event_reporter/metrics/metrics.go [hunk 3]
```diff
@@ -43,57 +45,54 @@ const (
...
+		[]string{"reporter_shard", "application", "error_in_learning_mode"},
 	)
-	erroredEventsCounter = prometheus.NewCounterVec(
+
+	droppedEventsCounter = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
-			Name: "cf_e_reporter_errored_events",
-			Help: "Total amount of errored events.",
+			Name: "codefresh_event_reporter_dropped_events_total",
+			Help: "Amount of dropped application events queue of taken shard.",
```
Comment (zarbis) [inline event_reporter/metrics/metrics.go:67]: "Amount of application events not accepted into the queue of a particular shard."

File: event_reporter/metrics/metrics.go [hunk 4]
```diff
@@ -43,57 +45,54 @@ const (
...
+		[]string{"reporter_shard", "metric_event_type", "error_type", "application"},
 	)
+
 	eventProcessingDurationHistogram = prometheus.NewHistogramVec(
 		prometheus.HistogramOpts{
-			Name:    "cf_e_reporter_event_processing_duration",
+			Name:    "codefresh_event_reporter_event_processing_duration",
 			Help:    "Application event processing duration.",
-			Buckets: []float64{0.1, 0.25, .5, 1, 2, 3, 4, 5, 7, 10, 15, 20},
+			Buckets: []float64{0.25, .5, 1, 2, 3, 5, 10, 20, 50},
```
Comment (zarbis) [inline event_reporter/metrics/metrics.go]: Let's leave just `{0.25, .5, 1, 2, 5, 10, 50}`

**Review by zarbis — APPROVED** [review]

**Pull request merged**